addons/web/static/src/core/ui/ui_service.js

```diff
@@ -31,13 +33,42 @@ export function useActiveElement(refName) {
         (el) => {
             if (el) {
                 uiService.activateElement(el);
+                trapFocus(el);
                 return () => uiService.deactivateElement(el);
             }
         },
         () => [owner.el]
     );
 }
 
+function trapFocus(element) {
+    const focusableEls = getTabableElements(element);
+    const firstFocusableEl = focusableEls[0];
+    const lastFocusableEl = focusableEls[focusableEls.length - 1];
+
+    element.addEventListener("keydown", function (e) {
+        const keyPressed = getActiveHotkey(e);
+        switch (keyPressed) {
+            case "tab":
+                if (document.activeElement === lastFocusableEl) {
+                    firstFocusableEl.focus();
+                    e.preventDefault();
```

Comment:
The strategy we currently agree on is to stop an event once it has been treated and you now no other behavior should get applied.
You should still prevent default though.

If you don't stop the event, any registration made in the hotkey service about these hotkeys will get applied as well.